contributor/outline.md

```diff
@@ -67,12 +68,13 @@ Outline for this section inspired by the [Trusted Committer introduction video].
     - Feedback from the host team is not any kind of personal criticism to you.
     Treat and value it as data to help your contribution become all the better.
     - If the host team decides not to accept a contribution, don't take it personally.
+    Work with them to understand their reasoning.
```

Comment:
Great, shouldn't this be indented to the level of line 70 though?
```suggestion
      - Work with them to understand their reasoning.
```

Reply:
It's meant to just be another sentence on the previous bullet.  You can see that, when rendered, it all forms a paragraph together.

Reply:
True, my bad, my mental Markdown Renderer apparently wasn't up to speed. ;) 
Closing this one then.